### DISCUSSION TITLE
Would this be possible, or any better way?

### DISCUSSION THREAD

**Discussion by ozgurerdogan** [psy0rz/zfs_autobackup#210] [General]
On some proxmox nodes, I need to snapshot some vms more often than others. They might be on same pool.

Can I create two zfs propery like:
autobackup:every-hour and also autobackup:every-day for individiual datasets use sync them so? Or any better way?

**Comment by norsemangrey** [comment]
I think probably using custom properties like that would be the way to go. I'm doing something similar on my server where I want to snapshot some datasets with different frequency so I have properties like `snapshot-pri-1:main-server`, `snapshot-pri-2:main-server`.

**Reply by psy0rz** [reply]
yes this is the way to go.

**Reply by ozgurerdogan** [reply]
Thanks for confirm.

**Comment by ozgurerdogan** [comment]
Do you use that properties on pool level or dataset level individually?

**Reply by norsemangrey** [reply]
I use them on individual datasets. I have quite a few so I use a script I made to make it easier (it's linked in a post further down if you want to try it out).

**Comment by ozgurerdogan** [comment]
I already tried that script but did not find time to play. But seems very usable. It is in my plans. Thanks for your time.
